normandy/settings.py

```diff
@@ -148,6 +148,40 @@ def MIDDLEWARE_CLASSES(self):
         """
         return Core.MIDDLEWARE_CLASSES + self.EXTRA_MIDDLEWARE_CLASSES
 
+    LOGGING_USE_JSON = values.BooleanValue(False)
+
+    def LOGGING(self):
+        return {
+            'version': 1,
+            'disable_existing_loggers': True,
+            'formatters': {
+                'json': {
+                    '()': 'mozilla_cloud_services_logger.formatters.JsonLogFormatter',
+                    'logger_name': 'normandy',
+                },
+                'development': {
+                    'format': '%(levelname)s %(asctime)s %(name)s %(message)s',
+                }, },
+            'handlers': {
+                'console': {
+                    'level': 'DEBUG',
+                    'class': 'logging.StreamHandler',
+                    'formatter': 'json' if self.LOGGING_USE_JSON else 'development',
+                },
+            },
+            'root': {
+                'handlers': ['console'],
```

Comment:
Maybe make the setting `LOGGING_HANDLERS` and use it here and below? Would allow for a bit more flexibility.